src/Sentry/Exceptions/SentryDebugImage.cs

```diff
@@ -0,0 +1,94 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+using Sentry.Extensibility;
+using Sentry.Internal.Extensions;
+
+namespace Sentry
+{
+    /// <summary>
+    /// The Sentry Debug Meta Images interface.
+    /// </summary>
+    /// <see href="https://develop.sentry.dev/sdk/event-payloads/debugmeta#debug-images"/>
+    public sealed class SentryDebugImage : IJsonSerializable
```

Comment:
Unlikely that `DebugImage` would conflict with default types (when `ImplicitUsings` is used) I'd say we can go with `DebugImage`:

```suggestion
    public sealed class DebugImage : IJsonSerializable
```